### DISCUSSION TITLE
Teslamate Battery Health Dashboard no data

### DISCUSSION THREAD

**Discussion by farhan0305** [teslamate-org/teslamate#4302] [Q&A]
Hello Community , 

I have been using TeslaMate for over 3 months now to record my Tesla's data , and all dashboards ( Drives ,Drive stats, Efficiency, Charges etc) are working as expected with the only exception of the Battery Health dashboard  where the Battery capacity , Degradation , Health widgets display NO DATA with the following error : 

"Status: 500. Message: db query error: pq: invalid input syntax for type json"

I have over 100 drives recored with over 65 charges logged.   I also tried downloading an Alternate Battery Health dashboard by Carlos Cuezva (https://github.com/CarlosCuezva/dashboards-Grafana-Teslamate/blob/main/dashboards/battery-health.json)  that I found here on GitHub and that seems to be working . 

Is there a way I can fix the default health dashboard please ?

![SmartSelect_20241024_084630_Firefox](https://github.com/user-attachments/assets/fdb88e50-2de1-4dc2-ad57-ece852f031a4)
![SmartSelect_20241024_084718_Firefox](https://github.com/user-attachments/assets/c918465c-b32d-462d-92a4-69384a9f094a)

**Comment by jovelang** [comment]
Have you solved it? I have the same problem
